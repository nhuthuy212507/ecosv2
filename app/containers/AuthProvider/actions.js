/*
 *
 * AuthProvider actions
 *
 */

import {
  GET_PROFILE,
  GET_PROFILE_SUCCESS,
  GET_PROFILE_ERROR,
} from './constants';

export function getProfile() {
  return {
    type: GET_PROFILE,
  };
}

export function getProfileSuccess(response) {
  return {
    type: GET_PROFILE_SUCCESS,
    response,
  };
}

export function getProfileError(error) {
  return {
    type: GET_PROFILE_ERROR,
    error,
  };
}
