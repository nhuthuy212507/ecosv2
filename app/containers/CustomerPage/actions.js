/*
 *
 * CustomerPage actions
 *
 */

import {
  GET_LIST_CUSTOMER,
  GET_LIST_CUSTOMER_SUCCESS,
  GET_LIST_CUSTOMER_ERROR,
  GET_OPTIONS_CUSTOMER_REFERENCE,
  GET_OPTIONS_CUSTOMER_REFERENCE_SUCCESS,
  GET_OPTIONS_CUSTOMER_REFERENCE_ERROR,
  GET_OPTIONS_CREATED_BY,
  GET_OPTIONS_CREATED_BY_SUCCESS,
  GET_OPTIONS_CREATED_BY_ERROR,
  ADD_CUSTOMER,
  ADD_CUSTOMER_SUCCESS,
  ADD_CUSTOMER_ERROR,
  EDIT_CUSTOMER,
  EDIT_CUSTOMER_SUCCESS,
  EDIT_CUSTOMER_ERROR,
  GET_CUSTOMER_EDITTING,
  GET_CUSTOMER_EDITTING_SUCCESS,
  GET_CUSTOMER_EDITTING_ERROR,
  GET_OPTIONS_CITY_WAREHOUSE_ADDRESS,
  GET_OPTIONS_CITY_WAREHOUSE_ADDRESS_SUCCESS,
  GET_OPTIONS_CITY_WAREHOUSE_ADDRESS_ERROR,
  GET_OPTIONS_DISTRICT_WAREHOUSE_ADDRESS,
  GET_OPTIONS_DISTRICT_WAREHOUSE_ADDRESS_SUCCESS,
  GET_OPTIONS_DISTRICT_WAREHOUSE_ADDRESS_ERROR,
  GET_OPTIONS_WARD_WAREHOUSE_ADDRESS,
  GET_OPTIONS_WARD_WAREHOUSE_ADDRESS_SUCCESS,
  GET_OPTIONS_WARD_WAREHOUSE_ADDRESS_ERROR,
  ADD_WAREHOUSE_ADDRESS,
  ADD_WAREHOUSE_ADDRESS_SUCCESS,
  ADD_WAREHOUSE_ADDRESS_ERROR,
  GET_LIST_WAREHOUSE_ADDRESS,
  GET_LIST_WAREHOUSE_ADDRESS_SUCCESS,
  GET_LIST_WAREHOUSE_ADDRESS_ERROR,
  EDIT_WAREHOUSE_ADDRESS,
  EDIT_WAREHOUSE_ADDRESS_SUCCESS,
  EDIT_WAREHOUSE_ADDRESS_ERROR,
} from './constants';

export function getListCustomer(params, actions = false) {
  return {
    type: GET_LIST_CUSTOMER,
    params,
    actions,
  };
}

export function getListCustomerSuccess(response) {
  return {
    type: GET_LIST_CUSTOMER_SUCCESS,
    response,
  };
}

export function getListCustomerError(error) {
  return {
    type: GET_LIST_CUSTOMER_ERROR,
    error,
  };
}

export function getOptionsCustomerReference(params) {
  return {
    type: GET_OPTIONS_CUSTOMER_REFERENCE,
    params,
  };
}

export function getOptionsCustomerReferenceSuccess(response) {
  return {
    type: GET_OPTIONS_CUSTOMER_REFERENCE_SUCCESS,
    response,
  };
}

export function getOptionsCustomerReferenceError(error) {
  return {
    type: GET_OPTIONS_CUSTOMER_REFERENCE_ERROR,
    error,
  };
}

export function getOptionsCreatedBy(params) {
  return {
    type: GET_OPTIONS_CREATED_BY,
    params,
  };
}

export function getOptionsCreatedBySuccess(response) {
  return {
    type: GET_OPTIONS_CREATED_BY_SUCCESS,
    response,
  };
}

export function getOptionsCreatedByError(error) {
  return {
    type: GET_OPTIONS_CREATED_BY_ERROR,
    error,
  };
}

export function addCustomer(data, actions = false) {
  return {
    type: ADD_CUSTOMER,
    data,
    actions,
  };
}

export function addCustomerSuccess() {
  return {
    type: ADD_CUSTOMER_SUCCESS,
  };
}

export function addCustomerError(error) {
  return {
    type: ADD_CUSTOMER_ERROR,
    error,
  };
}

export function getCustomerEditting(id) {
  return {
    type: GET_CUSTOMER_EDITTING,
    id,
  };
}

export function getCustomerEdittingSuccess(response) {
  return {
    type: GET_CUSTOMER_EDITTING_SUCCESS,
    response,
  };
}

export function getCustomerEdittingError(error) {
  return {
    type: GET_CUSTOMER_EDITTING_ERROR,
    error,
  };
}

export function editCustomer(id, data, actions = false) {
  return {
    type: EDIT_CUSTOMER,
    id,
    data,
    actions,
  };
}

export function editCustomerSuccess() {
  return {
    type: EDIT_CUSTOMER_SUCCESS,
  };
}

export function editCustomerError(error) {
  return {
    type: EDIT_CUSTOMER_ERROR,
    error,
  };
}

export function getOptionsCityWarehouseAddress(params) {
  return {
    type: GET_OPTIONS_CITY_WAREHOUSE_ADDRESS,
    params,
  };
}

export function getOptionsCityWarehouseAddressSuccess(response) {
  return {
    type: GET_OPTIONS_CITY_WAREHOUSE_ADDRESS_SUCCESS,
    response,
  };
}

export function getOptionsCityWarehouseAddressError(error) {
  return {
    type: GET_OPTIONS_CITY_WAREHOUSE_ADDRESS_ERROR,
    error,
  };
}

export function getOptionsDistrictWarehouseAddress(params) {
  return {
    type: GET_OPTIONS_DISTRICT_WAREHOUSE_ADDRESS,
    params,
  };
}

export function getOptionsDistrictWarehouseAddressSuccess(response) {
  return {
    type: GET_OPTIONS_DISTRICT_WAREHOUSE_ADDRESS_SUCCESS,
    response,
  };
}

export function getOptionsDistrictWarehouseAddressError(error) {
  return {
    type: GET_OPTIONS_DISTRICT_WAREHOUSE_ADDRESS_ERROR,
    error,
  };
}

export function getOptionsWardWarehouseAddress(params) {
  return {
    type: GET_OPTIONS_WARD_WAREHOUSE_ADDRESS,
    params,
  };
}

export function getOptionsWardWarehouseAddressSuccess(response) {
  return {
    type: GET_OPTIONS_WARD_WAREHOUSE_ADDRESS_SUCCESS,
    response,
  };
}

export function getOptionsWardWarehouseAddressError(error) {
  return {
    type: GET_OPTIONS_WARD_WAREHOUSE_ADDRESS_ERROR,
    error,
  };
}

export function addWarehouseAddress(data, actions = false) {
  return {
    type: ADD_WAREHOUSE_ADDRESS,
    data,
    actions,
  };
}

export function addWarehouseAddressSuccess() {
  return {
    type: ADD_WAREHOUSE_ADDRESS_SUCCESS,
  };
}

export function addWarehouseAddressError(error) {
  return {
    type: ADD_WAREHOUSE_ADDRESS_ERROR,
    error,
  };
}

export function getListWarehouseAddress(params, actions = false) {
  return {
    type: GET_LIST_WAREHOUSE_ADDRESS,
    params,
    actions,
  };
}

export function getListWarehouseAddressSuccess(response) {
  return {
    type: GET_LIST_WAREHOUSE_ADDRESS_SUCCESS,
    response,
  };
}

export function getListWarehouseAddressError(error) {
  return {
    type: GET_LIST_WAREHOUSE_ADDRESS_ERROR,
    error,
  };
}

export function editWarehouseAddress(data, actions = false) {
  return {
    type: EDIT_WAREHOUSE_ADDRESS,
    data,
    actions,
  };
}

export function editWarehouseAddressSuccess() {
  return {
    type: EDIT_WAREHOUSE_ADDRESS_SUCCESS,
  };
}

export function editWarehouseAddressError(error) {
  return {
    type: EDIT_WAREHOUSE_ADDRESS_ERROR,
    error,
  };
}

export function deleteWarehouseAddress(data, actions = false) {
  return {
    type: EDIT_WAREHOUSE_ADDRESS,
    data,
    actions,
  };
}

export function deleteWarehouseAddressSuccess() {
  return {
    type: EDIT_WAREHOUSE_ADDRESS_SUCCESS,
  };
}

export function deleteWarehouseAddressError(error) {
  return {
    type: EDIT_WAREHOUSE_ADDRESS_ERROR,
    error,
  };
}
