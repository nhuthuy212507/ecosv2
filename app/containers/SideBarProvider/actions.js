/*
 *
 * SideBarProvider actions
 *
 */

import {
  CHANGE_DESKTOP_SIDEBAR_VISIBILITY,
  CHANGE_MOBILE_SIDEBAR_VISIBILITY,
} from './constants';

export function changeDesktopSidebarVisibility() {
  return {
    type: CHANGE_DESKTOP_SIDEBAR_VISIBILITY,
  };
}

export function changeMobileSidebarVisibility() {
  return {
    type: CHANGE_MOBILE_SIDEBAR_VISIBILITY,
  };
}
