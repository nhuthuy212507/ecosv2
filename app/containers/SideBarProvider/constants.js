/*
 *
 * SideBarProvider constants
 *
 */

export const CHANGE_DESKTOP_SIDEBAR_VISIBILITY =
  'app/SideBarProvider/CHANGE_DESKTOP_SIDEBAR_VISIBILITY';

export const CHANGE_MOBILE_SIDEBAR_VISIBILITY =
  'app/SideBarProvider/CHANGE_MOBILE_SIDEBAR_VISIBILITY';
