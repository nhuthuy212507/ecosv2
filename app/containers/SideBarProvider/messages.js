/*
 * SideBarProvider Messages
 *
 * This contains all the text for the SideBarProvider container.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.containers.SideBarProvider';

export default defineMessages({
  dashboard: {
    id: `${scope}.dashboard`,
    defaultMessage: 'Dashboard',
  },
  customer: {
    id: `${scope}.customer`,
    defaultMessage: 'Customer',
  },
});
