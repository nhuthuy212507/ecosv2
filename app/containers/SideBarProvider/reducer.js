/*
 *
 * SideBarProvider reducer
 *
 */
import produce from 'immer';
import {
  CHANGE_DESKTOP_SIDEBAR_VISIBILITY,
  CHANGE_MOBILE_SIDEBAR_VISIBILITY,
} from './constants';

export const initialState = {
  collapse: false,
  show: false,
};

/* eslint-disable default-case, no-param-reassign */
const sideBarProviderReducer = (state = initialState, action) =>
  produce(state, draft => {
    switch (action.type) {
      case CHANGE_DESKTOP_SIDEBAR_VISIBILITY:
        draft.collapse = !state.collapse;
        break;
      case CHANGE_MOBILE_SIDEBAR_VISIBILITY:
        draft.show = !state.show;
        break;
    }
  });

export default sideBarProviderReducer;
