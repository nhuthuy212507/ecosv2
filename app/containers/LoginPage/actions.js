/*
 *
 * LoginPage actions
 *
 */

import { LOGIN, LOGIN_SUCCESS, LOGIN_ERROR } from './constants';

export function login(user) {
  return {
    type: LOGIN,
    user,
  };
}

export function loginSuccess(token) {
  return {
    type: LOGIN_SUCCESS,
    token,
  };
}

export function loginError(error) {
  return {
    type: LOGIN_ERROR,
    error,
  };
}
