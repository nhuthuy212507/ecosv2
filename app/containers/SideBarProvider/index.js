/**
 *
 * SideBarProvider
 *
 */

import React from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import { createStructuredSelector } from 'reselect';
import { compose } from 'redux';

import { useInjectReducer } from 'utils/injectReducer';
import {
  Wrapper,
  ButtonBack,
  Scrollbar,
  SideBarDesktop,
  SideBarMobile,
} from 'components/SideBar';
import SideBarContent from './sidebarContent';
import { makeSelectCollapse, makeSelectShow } from './selectors';
import { changeMobileSidebarVisibility } from './actions';
import reducer from './reducer';

export function SideBarProvider({
  collapse,
  show,
  onChangeMobileSidebarVisibility,
}) {
  useInjectReducer({ key: 'sidebar', reducer });

  return (
    <Wrapper collapse={collapse} show={show}>
      <ButtonBack show={show} onClick={onChangeMobileSidebarVisibility} />
      <Scrollbar>
        <SideBarDesktop>
          <SideBarContent collapse={collapse} />
        </SideBarDesktop>
        <SideBarMobile>
          <SideBarContent collapse={collapse} />
        </SideBarMobile>
      </Scrollbar>
    </Wrapper>
  );
}

SideBarProvider.propTypes = {
  collapse: PropTypes.bool.isRequired,
  show: PropTypes.bool.isRequired,
  onChangeMobileSidebarVisibility: PropTypes.func.isRequired,
};

const mapStateToProps = createStructuredSelector({
  collapse: makeSelectCollapse(),
  show: makeSelectShow(),
});

export function mapDispatchToProps(dispatch) {
  return {
    onChangeMobileSidebarVisibility: () =>
      dispatch(changeMobileSidebarVisibility()),
  };
}

const withConnect = connect(
  mapStateToProps,
  mapDispatchToProps,
);

export default compose(withConnect)(SideBarProvider);
