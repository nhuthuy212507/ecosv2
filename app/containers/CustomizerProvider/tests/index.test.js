/**
 * Test the HomePage
 */

import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';
import { browserHistory } from 'react-router-dom';

import { changeDesktopSidebarVisibility } from 'containers/SideBarProvider/actions';
import { CustomizerProvider, mapDispatchToProps } from '../index';
import { toggleTheme } from '../actions';
import configureStore from '../../../configureStore';

describe('<CustomizerProvider />', () => {
  let store;

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('should render and match the snapshot', () => {
    const toggleSpy = jest.fn();
    const changeDesktopSidebarVisibilitySpy = jest.fn();
    const {
      container: { firstChild },
    } = render(
      <Provider store={store}>
        <IntlProvider locale="en">
          <CustomizerProvider
            isDarkTheme={false}
            collapse={false}
            onToggleTheme={toggleSpy}
            onChangeDesktopSidebarVisibility={changeDesktopSidebarVisibilitySpy}
          />
        </IntlProvider>
      </Provider>,
    );
    expect(firstChild).toMatchSnapshot();
  });

  describe('mapDispatchToProps', () => {
    describe('onToggleTheme', () => {
      it('should be injected', () => {
        const dispatch = jest.fn();
        const result = mapDispatchToProps(dispatch);
        expect(result.onToggleTheme).toBeDefined();
      });

      it('should dispatch onToggleTheme when called', () => {
        const dispatch = jest.fn();
        const result = mapDispatchToProps(dispatch);
        result.onToggleTheme();
        expect(dispatch).toHaveBeenCalledWith(toggleTheme());
      });
    });

    describe('onChangeDesktopSidebarVisibility', () => {
      it('should be injected', () => {
        const dispatch = jest.fn();
        const result = mapDispatchToProps(dispatch);
        expect(result.onChangeDesktopSidebarVisibility).toBeDefined();
      });

      it('should dispatch onChangeDesktopSidebarVisibility when called', () => {
        const dispatch = jest.fn();
        const result = mapDispatchToProps(dispatch);
        result.onChangeDesktopSidebarVisibility();
        expect(dispatch).toHaveBeenCalledWith(changeDesktopSidebarVisibility());
      });
    });
  });
});
