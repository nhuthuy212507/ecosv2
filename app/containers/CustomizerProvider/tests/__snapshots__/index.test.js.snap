// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomizerProvider /> should render and match the snapshot 1`] = `
<div
  class="Wrapper-sc-1515fau-0 cyWzjs"
>
  <button
    class="CustomizerBtn-ipp7qd-0 jFVNiU"
  >
    <i
      aria-label="icon: setting"
      class="anticon anticon-setting CustomizerBtnIcon-sc-1brw7l1-0 hFJPKl"
    >
      <svg
        aria-hidden="true"
        class=""
        data-icon="setting"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M924.8 625.7l-65.5-56c3.1-19 4.7-38.4 4.7-57.8s-1.6-38.8-4.7-57.8l65.5-56a32.03 32.03 0 0 0 9.3-35.2l-.9-2.6a443.74 443.74 0 0 0-79.7-137.9l-1.8-2.1a32.12 32.12 0 0 0-35.1-9.5l-81.3 28.9c-30-24.6-63.5-44-99.7-57.6l-15.7-85a32.05 32.05 0 0 0-25.8-25.7l-2.7-.5c-52.1-9.4-106.9-9.4-159 0l-2.7.5a32.05 32.05 0 0 0-25.8 25.7l-15.8 85.4a351.86 351.86 0 0 0-99 57.4l-81.9-29.1a32 32 0 0 0-35.1 9.5l-1.8 2.1a446.02 446.02 0 0 0-79.7 137.9l-.9 2.6c-4.5 12.5-.8 26.5 9.3 35.2l66.3 56.6c-3.1 18.8-4.6 38-4.6 57.1 0 19.2 1.5 38.4 4.6 57.1L99 625.5a32.03 32.03 0 0 0-9.3 35.2l.9 2.6c18.1 50.4 44.9 96.9 79.7 137.9l1.8 2.1a32.12 32.12 0 0 0 35.1 9.5l81.9-29.1c29.8 24.5 63.1 43.9 99 57.4l15.8 85.4a32.05 32.05 0 0 0 25.8 25.7l2.7.5a449.4 449.4 0 0 0 159 0l2.7-.5a32.05 32.05 0 0 0 25.8-25.7l15.7-85a350 350 0 0 0 99.7-57.6l81.3 28.9a32 32 0 0 0 35.1-9.5l1.8-2.1c34.8-41.1 61.6-87.5 79.7-137.9l.9-2.6c4.5-12.3.8-26.3-9.3-35zM788.3 465.9c2.5 15.1 3.8 30.6 3.8 46.1s-1.3 31-3.8 46.1l-6.6 40.1 74.7 63.9a370.03 370.03 0 0 1-42.6 73.6L721 702.8l-31.4 25.8c-23.9 19.6-50.5 35-79.3 45.8l-38.1 14.3-17.9 97a377.5 377.5 0 0 1-85 0l-17.9-97.2-37.8-14.5c-28.5-10.8-55-26.2-78.7-45.7l-31.4-25.9-93.4 33.2c-17-22.9-31.2-47.6-42.6-73.6l75.5-64.5-6.5-40c-2.4-14.9-3.7-30.3-3.7-45.5 0-15.3 1.2-30.6 3.7-45.5l6.5-40-75.5-64.5c11.3-26.1 25.6-50.7 42.6-73.6l93.4 33.2 31.4-25.9c23.7-19.5 50.2-34.9 78.7-45.7l37.9-14.3 17.9-97.2c28.1-3.2 56.8-3.2 85 0l17.9 97 38.1 14.3c28.7 10.8 55.4 26.2 79.3 45.8l31.4 25.8 92.8-32.9c17 22.9 31.2 47.6 42.6 73.6L781.8 426l6.5 39.9zM512 326c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm79.2 255.2A111.6 111.6 0 0 1 512 614c-29.9 0-58-11.7-79.2-32.8A111.6 111.6 0 0 1 400 502c0-29.9 11.7-58 32.8-79.2C454 401.6 482.1 390 512 390c29.9 0 58 11.6 79.2 32.8A111.6 111.6 0 0 1 624 502c0 29.9-11.7 58-32.8 79.2z"
        />
      </svg>
    </i>
  </button>
  <div
    class="CustomizerWrap-sc-1lk3wih-0 csCNvD"
  >
    <div
      class="CustomizerTitle-sc-143qm2z-0 bBLBvQ"
    >
      <h5>
        <span>
          Theme Customizer
        </span>
      </h5>
      <div
        class="CloseButton-sc-1nbzn33-0 byWECT"
      >
        <i
          aria-label="icon: close-circle"
          class="anticon anticon-close-circle"
        >
          <svg
            aria-hidden="true"
            class=""
            data-icon="close-circle"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M685.4 354.8c0-4.4-3.6-8-8-8l-66 .3L512 465.6l-99.3-118.4-66.1-.3c-4.4 0-8 3.5-8 8 0 1.9.7 3.7 1.9 5.2l130.1 155L340.5 670a8.32 8.32 0 0 0-1.9 5.2c0 4.4 3.6 8 8 8l66.1-.3L512 564.4l99.3 118.4 66 .3c4.4 0 8-3.5 8-8 0-1.9-.7-3.7-1.9-5.2L553.5 515l130.1-155c1.2-1.4 1.8-3.3 1.8-5.2z"
            />
            <path
              d="M512 65C264.6 65 64 265.6 64 513s200.6 448 448 448 448-200.6 448-448S759.4 65 512 65zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
            />
          </svg>
        </i>
      </div>
    </div>
    <div
      class="CustomizerToggle-sc-12p2dxf-0 cVOFOA"
    >
      <button
        aria-checked="false"
        class="ant-switch"
        role="switch"
        type="button"
      >
        <span
          class="ant-switch-inner"
        />
      </button>
      <span>
        Dark Theme
      </span>
    </div>
    <div
      class="CustomizerToggle-sc-12p2dxf-0 cVOFOA"
    >
      <button
        aria-checked="false"
        class="ant-switch"
        role="switch"
        type="button"
      >
        <span
          class="ant-switch-inner"
        />
      </button>
      <span>
        Collapsed Menu
      </span>
    </div>
  </div>
</div>
`;
